services/core-api/app/api/now_applications/models/activity_summary/camp.py

```diff
@@ -17,21 +17,42 @@ class Camp(ActivitySummaryBase):
     activity_summary_id = db.Column(
         db.Integer, db.ForeignKey('activity_summary.activity_summary_id'), primary_key=True)
 
-    camp_name = db.Column(db.String)
-    camp_number_people = db.Column(db.String)
-    camp_number_structures = db.Column(db.String)
-    has_fuel_stored = db.Column(db.Boolean, nullable=False, server_default=FetchedValue())
-    has_fuel_stored_in_bulk = db.Column(db.Boolean, nullable=False, server_default=FetchedValue())
-    has_fuel_stored_in_barrels = db.Column(
-        db.Boolean, nullable=False, server_default=FetchedValue())
+    health_authority_notified = db.Column(db.Boolean, nullable=True)
```

Comment:
No need to specify "nullable=True" across the board as it is the assumed value columns.

Reply:
removed.